vcr/oidc4vci/types.go

```diff
@@ -84,7 +84,7 @@ type CredentialOffer struct {
 	// Credentials defines the credentials offered by the issuer to the wallet.
 	Credentials []map[string]interface{} `json:"credentials"`
 	// Grants defines the grants offered by the issuer to the wallet.
-	Grants []map[string]interface{} `json:"grants"`
+	Grants map[string]interface{} `json:"grants"`
```

Comment:
spec says it is optional

Reply:
For the sake of simple Golang structure (nil checking for pointers), probably easiest to keep it non-optional here? (then it is returned as empty map)